lib/legacy_mempool.go

```diff
@@ -2131,13 +2131,24 @@ func _computeBitcoinExchangeFields(params *DeSoParams,
 
 func ConnectTxnAndComputeTransactionMetadata(
 	txn *MsgDeSoTxn, utxoView *UtxoView, blockHash *BlockHash,
-	blockHeight uint32, blockTimestampNanoSecs int64, txnIndexInBlock uint64) (*TransactionMetadata, error) {
+	blockHeight uint32, blockTimestampNanoSecs int64, txnIndexInBlock uint64, connects bool) (*TransactionMetadata, error) {
 
 	totalNanosPurchasedBefore := utxoView.NanosPurchased
 	usdCentsPerBitcoinBefore := utxoView.GetCurrentUSDCentsPerBitcoin()
-	utxoOps, totalInput, totalOutput, fees, err := utxoView._connectTransaction(
-		txn, txn.Hash(), blockHeight, blockTimestampNanoSecs, false, false,
-	)
+
+	var utxoOps []*UtxoOperation
```

Comment:
Should we move this out of legacy_mempool.go and into a file more specific to TxIndex?